ckanext/who_afro/blueprints/errors.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from flask import Blueprint, render_template
+
+log = logging.getLogger(__name__)
+
+blueprint = Blueprint(
+    "errors", __name__, url_prefix="/errors"
```

Comment:
Not entirely sure this is an "error" though?  Did wonder whether it could go into the "overview" blueprint.  But not sure. If you feel strongly this is the best way, happy to leave as is.